packages/@dcl/sdk-commands/src/logic/scene-validations.ts

```diff
@@ -97,6 +97,25 @@ export function assertValidScene(
 https://docs.decentraland.org/creator/development-guide/scene-metadata/#scene-title-description-and-image`
     )
   }
+
+  return scene
+}
+
+// TMP HACK: we should do this in unity-renderer, but for now we can patch it here...
+// remove when not needed and everything should be fine
+function patchSceneJson(scene: Scene): Scene {
```

Comment:
this permissions should always be enabled for non-px scenes. Talked with @pravusjif and we are handling them client side and, for the time being, we are patching it here until we can fix a bug with the scene data on unity-renderer.
Writing them to the creator's `scene.json` would show changes on diff and generate confusion's.

Reply:
Basically it's injecting the missing permissions for non-px scenes, so that the when the client asks for those permissions internally it finds them, in the case of a non-px scene.

The `isPortableExperience` bool that we have on `ParcelScene` in the client is not working at all, in the future we can debug that bool so that it works and skip those permission checks inside the client for non-px scenes.

Reply:
Yes but this will only work on the Preview mode since we inject those only on the start.
When you deploy the scene, those permissions wont be there for the client. Is that expected ? 🤔 
Thats why i suggest to update the scene.json.
Other condition would be to set this in kernel instead of here. Like we do for fetch & websockets.

Reply:
closed in favor of: https://github.com/decentraland/unity-renderer/pull/5888